### ISSUE TITLE
Bad location of the Commented version line in presence of COBOL Options.

### ISSUE THREAD

**Issue by mayanje** [TypeCobolTeam/TypeCobol#958]
In the following generated Code the TYpeCobol version appears before de CBL option.
```cobol
      *TypeCobol_Version:0.1(alpha)
       CBL ARITH(EXTEND)
       IDENTIFICATION DIVISION.
       PROGRAM-ID. DVZXERRM.
       DATA DIVISION.
       WORKING-STORAGE SECTION.
       PROCEDURE DIVISION.
       END PROGRAM DVZXERRM.
```
See also #932.

**Comment by smedilol** [comment]
Is this issue resolved with #932 ?

**Comment by mayanje** [comment]
No this is a new issue because we must consider the case when the generated code is not given to TypeCobol.Transform.
#932 filters TypeCobol version comment.
